### PR TITLE
Cockpit 217

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-217.html

### CODE REVIEW

File: _posts/2020-04-15-cockpit-217.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags: cockpit linux
+slug: cockpit-217
+category: release
+summary: Cockpit with improved ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.
+
+### More details about Insights rule hits
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:14]: Please prefix with the page, "Overview:".

File: _posts/2020-04-15-cockpit-217.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 217
+author: mvo
+date: 2020-04-15
+tags: cockpit linux
+slug: cockpit-217
+category: release
+summary: Cockpit with improved ...
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:8]: TODO :)

File: _posts/2020-04-15-cockpit-217.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+If your machine is connected to Red Hat Insights, the Health card on
+the Overview page will now show more details.
+
+![Insights details](/images/health-insights.png)
+
+### Buttons are now left aligned
+
+In accordance with Patternfly (XXX-LINK-XXX), the buttons in all modal
+dialogs have been left aligned.
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:24]: And more importantly, Cancel and other secondary buttons have been de-emphasized to be link buttons. (That's not release note ready text, of course).

File: _posts/2020-04-15-cockpit-217.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+![New button order](/images/new-button-order.png)
+
+### Sending keys to VMs
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:28]: Please prefix with "Machines:". Also, the general feature has existed for a while, so maybe "Send VT switching keys to VMs"?

File: _posts/2020-04-15-cockpit-217.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+It is now possible to use a menu to send a number of special keys to a
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:30]: ... send keys Ctrl+Alt+F1 to F12 to...

File: _posts/2020-04-15-cockpit-217.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+If your machine is connected to Red Hat Insights, the Health card on
+the Overview page will now show more details.
+
+![Insights details](/images/health-insights.png)
+
+### Dialog buttons have been improved
+
+Following the lead of
+[Patternfly](https://blog.patternfly.org/patternfly/button-placement-on-forms/),
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:24]: It's spelled "PatternFly".

File: _posts/2020-04-15-cockpit-217.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+the buttons in all modal dialogs have been changed to improve
+useability and accessibility.
```
Comment (martinpitt) [inline _posts/2020-04-15-cockpit-217.md:26]: I think it's "usability".

**Comment by martinpitt** [comment]
bodhi URLs are available now: https://bodhi.fedoraproject.org/updates/?packages=cockpit

File: _posts/2020-04-15-cockpit-217.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Done.

File: _posts/2020-04-15-cockpit-217.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Fixed.

File: _posts/2020-04-15-cockpit-217.md (hunk 4 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Ok! Fixed.

File: _posts/2020-04-15-cockpit-217.md (hunk 5 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Fixed.

File: _posts/2020-04-15-cockpit-217.md (hunk 6 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Fixed.

File: _posts/2020-04-15-cockpit-217.md (hunk 7 above)
Comment (mvollmer) [reply _posts/2020-04-15-cockpit-217.md]: Fixed.

File: _posts/2020-04-15-cockpit-217.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+summary: Cockpit with improved Insights details, dialog buttons, and VM consoles
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.
+
+### Overview: More details about Insights rule hits
+
+If your machine is connected to Red Hat Insights, the Health card on
+the Overview page will now show more details.
```
Comment (garrett) [inline _posts/2020-04-15-cockpit-217.md:17]:
```suggestion
When a machine is connected to Red Hat Insights, the Health card on
the Overview page now shows more details.
```

File: _posts/2020-04-15-cockpit-217.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+![Insights details](/images/health-insights.png)
+
+### Dialogs: Buttons have been improved
+
+Following the lead of
+[PatternFly](https://blog.patternfly.org/patternfly/button-placement-on-forms/),
+the buttons in all modal dialogs have been changed to improve
+usability and accessibility.
```
Comment (garrett) [inline _posts/2020-04-15-cockpit-217.md:26]:
```suggestion
Following [PatternFly's revised design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/buttons-and-links#button-placement), button placement in all modal dialogs has been changed [to improve usability and accessibility](https://blog.patternfly.org/patternfly/button-placement-on-forms/).
```

File: _posts/2020-04-15-cockpit-217.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+![New button order](/images/new-button-order.png)
+
+### Machines: Sending more keys to the console
+
+It is now possible to send a number of additional special keys to a
+virtual machine console, for switching Linux virtual consoles.
```
Comment (garrett) [inline _posts/2020-04-15-cockpit-217.md:33]:
```suggestion
Many more special keys can be sent to running virtual machines:

- Control+Alt+Delete is usually used for rebooting a computer. On Windows systems, it is also sometimes used for signing in.
- Control+Alt+Backspace is useful for force-quitting the X Window System.
- The rest of of these key combinations are for switching Linux virtual consoles.
```

**Pull request merged**